nominees/edunoss.json

```diff
@@ -0,0 +1,35 @@
+{
+  "name": "EduNOSS",
+  "aliases": [
+    "NOSS"
+  ],
+  "description": "A Linux OS with lots of educational applications to support teaching and learning of ICT and STEM in pre-tertiary educational institutions.",
+  "website": "https://www.aiti-kace.com.gh/noss",
+  "license": [
+    {
+      "spdx": "LGPL-2.1",
+      "licenseURL": "https://www.gnu.org/licenses/old-licenses/lgpl-2.1.en.html"
+    }
+  ],
+  "SDGs": [
+    {
+      "SDGNumber": 4,
+      "evidenceText": "This project seek to make available a computing platform with required educational applications, STEM Simulation tools and office productivity applications to enable pre-tertiary students learn and study ICT, Digital Skills and cultivate the habit of innovating for national development."
+    }
+  ],
+  "sectors": [],
+  "type": [
+    "software"
+  ],
+  "repositoryURL": "https://repos.nos.aiti-kace.com.gh/noss",
```

Comment:
We don't require a code repository per se, but access to the source code (where of course an open source code repository is the most convenient way to provide this). And it is a requirement of any OSI license to provide access to the source code, otherwise the project maintainer is infringing their own license 🤷‍♂️

So, in order to approve this we would need to require access to the source code, in whatever form they make it available.

@nathanbaleeta, I suggest you reach out to Frederick and clarify this point. You can refer him to this conversation, and he can clarify it here. Until then, I'm marking this nomination as blocked.

Reply:
Have reached out to Fredrick, will update this submission once we hear from him.

Reply:
Resolved in commit b9e0713